docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -212,14 +132,60 @@ The AESM service should be up and running. To confirm that, use:
 sudo systemctl status aesmd.service
 ```
 
-### Docker-enabled System
+#### Configuring Intel Quote Provider
 
-An easy way to install and run the AESM service on a [Docker](https://docs.docker.com/engine/)-enabled
+Intel Quote Provider (`libsgx-dcap-default-qpl`) needs to be configured to use
+either the Intel PCS, the PCCS of your cloud service provider, or your
+own PCCS. The configuration file is located at `/etc/sgx_default_qcnl.conf`.
+
+Make sure to always restart the `aesmd.service` after updating the
+configuration, via:
+
+```bash
+sudo systemctl restart aesmd.service
+```
+
+
+#### Intel PCS
+
+Using the Intel PCS is the simplest and most generic way, but it may be less
+reliable than using your cloud service provider (or your own PCCS).
```

Comment:
Also we should somehow mention that on some cloud providers (like the below) you can only use their PCCS because the VM guest doesn't have access to the platform manifest and so can't retrieve the PCK when key caching is disabled (which is the case on Azure).